app/controllers/application_controller.rb

```diff
@@ -54,8 +54,12 @@ def verify_private_forem
     return if self.class.module_parent.to_s == "Admin"
     return if user_signed_in? || Settings::UserExperience.public
 
+    @page = Page.find_by(landing_page: true)
```

Comment:
you could create a class method to load this inside `Page`, you could also add one to check existence:

```ruby
  def self.landing_page
    find_by(landing_page: true)
  end

  def self.landing_page_exists?
    exists?(landing_page: true)
  end
```

Reply:
Done! ✅